assets/global.js

```diff
@@ -712,10 +712,26 @@ class SlideshowComponent extends SliderComponent {
     this.slider.addEventListener('scroll', this.setSlideVisibility.bind(this));
     this.setSlideVisibility();
 
-    this.reducedMotion = window.matchMedia('(prefers-reduced-motion: reduce)');
-    this.reducedMotion.addEventListener('change', () => {
-      if (this.slider.getAttribute('data-autoplay') === 'true') this.setAutoPlay();
-    });
+    if (this.querySelector('.announcement-bar-slider')) {
+      this.announcementBarArrowButtonWasClicked = false;
+
+      this.desktopLayout = window.matchMedia('(min-width: 750px)');
+      this.reducedMotion = window.matchMedia('(prefers-reduced-motion: reduce)');
+
+      this.reducedMotion.addEventListener('change', () => {
+        if (this.slider.getAttribute('data-autoplay') === 'true') this.setAutoPlay();
+      });
+
+      this.desktopLayout.addEventListener('change', (event) => {
+        if (this.slider.getAttribute('data-autoplay') === 'true') this.setAutoPlay();
+      });
```

Comment:
I think you can do the same as we did with the buttons on those `change` events 🤔 

```javascript
[this.desktopLayout, this.reducedMotion].forEach((mediaQuery) => {
  mediaQuery.addEventListener('change', () => {
    if (this.slider.getAttribute('data-autoplay') === 'true') this.setAutoPlay();
  });
});
```

Reply:
Thanks Ludo.